tools/check-python.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# E501 line too long (N > 79 characters)
+# E731 do not assign a lambda expression, use a def
+flake8 --ignore=E501,E731 \
```

Comment:
No need to have a separate script just to run flake8, I'd just add this into the Makefile.